docs/source/getting-started/Setting-up-hypervisor-driver.adoc

```diff
@@ -1,20 +1,20 @@
-[[docker-machine-driver-install]]
-= Installing Docker Machine drivers
+[[set-up-hypervisor-driver]]
+= Setting up the hypervisor drivers
```

Comment:
-1 Same here, I am against this rename. We had this discussion and explicitly choose to use "Docker machine driver". If you enter this search string into Google, you get to the most appropriate documentation